RELEASE.md

```diff
@@ -0,0 +1,47 @@
+Release type: patch
+
+Add `process_result` to views for Django, Flask and ASGI. They can be overriden
+to provide a custom response and also to process results and errors.
+
+Django example:
+
+```python
+# views.py
+from strawberry.django.views import GraphQLView as BaseGraphQLView
+from strawberry.http import GraphQLHTTPResponse
+from strawberry.schema import ExecutionResult
+
+class GraphQLView(BaseGraphQLView):
+    def process_result(self, result: ExecutionResult) -> GraphQLHTTPResponse:
```

Comment:
It might be useful to pass the request here as well?

Reply:
Very good point! Will change 😄

Reply:
done! @jkimbo I've removed the request param from the flask view, since request is a global there 😊